onchain/rollups/contracts/dapp/ICartesiDApp.sol

```diff
@@ -46,14 +46,14 @@ interface ICartesiDApp {
     /// @param _payload The payload, which—in the case of Solidity contracts—encodes a function call
     /// @param _proof The proof used to validate the voucher against
     ///               a claim submitted by the current consensus contract
-    /// @return Whether the execution was successful or not
-    /// @dev On a successful execution, emits a `VoucherExecuted` event.
+    /// @dev No returned value, instead reverts on failure.
+    ///      On a successful execution, emits a `VoucherExecuted` event.
```

Comment:
We don't need to say that a function returns no value, as this information is already embedded in the signature. We should say, however, that the error is propagated as-is.

```suggestion
    /// @dev On a successful execution, emits a `VoucherExecuted` event.
    ///      On a failed execution, propagates the error data as-is.
```